Packs/PAN-OS/Integrations/Panorama/Panorama.py

```diff
@@ -6215,6 +6215,36 @@ def get_anti_spyware_best_practice_command():
     })
 
 
+def apply_dns_signature_policy_command(args: dict) -> CommandResults:
+    anti_spy_ware_name = args.get('anti_spyware_profile_name')
+    edl = args.get('dns_signature_source')
+    action = args.get('action')
+    packet_capture = args.get('packet_capture', 'disable')
+    params = {
+        'action': 'set',
+        'type': 'config',
+        'xpath': f"/config/devices/entry[@name='localhost.localdomain']/device-group/entry[@name='{DEVICE_GROUP}']"
+                 f"/profiles/spyware/entry[@name='{anti_spy_ware_name}']",
+        'key': API_KEY,
+        'element': '<botnet-domains>'
+                   f'<lists>'
+                   f'<entry name="{edl}"><packet-capture>{packet_capture}</packet-capture>'
+                   f'<action><{action}/></action></entry>'
+                   f'</lists>'
+                   f'</botnet-domains>'
+    }
+    result = http_request(
+        URL,
+        'POST',
+        params=params,
+    )
+    res_status = result.get('response', {}).get('@status')
```

Comment:
I suggest in case of an error, raise an exception which indicates what the error was. you should have it in the api.
There are many examples in the code. 

```suggestion
    res_status = result.get('response', {}).get('@status')
    if res_status == 'error':
        err_msg = result.get('response', {}).get('msg', {}).get('line'))
        raise DemistoException(f'Error: {err_msg})
```

Add this code section if it doesn't get caught in the http_request function.
if needed try to see what the api returns in case of error to fit the case here better. 
Other than that looks great, nice work!

Reply:
did you check it?

Reply:
@GuyAfik didnt see this comment i will check it and fixed the other comments if its ok will merge :)

Reply:
Done